resources/lib/api.py

```diff
@@ -270,7 +270,7 @@ def _findPlayback(playBackSets, lang=None, ask=False):
                     del config["ladder"]
                 if config not in options:
                     options.append(config)
-        options.sort(key=lambda x: quality.get(x.get("resolution", "sd")) or "")
+        options.sort(key=lambda x: str(quality.get(x.get("resolution", "sd")) or ""))
```

Comment:
```suggestion
        options.sort(key=lambda x: str(quality.get(x.get("resolution", "sd"), "")))
```

Both will work but this would be better.